modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3250,6 +3253,25 @@ public int compare(CodegenProperty one, CodegenProperty another) {
             }
         }
 
+        if (sortModelPropertiesByInheritanceFirst) {
```

Comment:
Should these options be exclusive?

Reply:
Good question.  I add a comment to clarify the behaviour